lib/config/configuration.go

```diff
@@ -110,6 +110,17 @@ type CommandLineFlags struct {
 
 	// AppPublicAddr is the public address of the application to proxy.
 	AppPublicAddr string
+
+	// DatabaseName is the name of the database to proxy.
+	DatabaseName string
+	// DatabaseProtocol is the type of the proxied database e.g. postgres or mysql.
+	DatabaseProtocol string
+	// DatabaseURI is the address to connect to the proxied database.
+	DatabaseURI string
+	// DatabaseCACertFile is the database CA cert path.
+	DatabaseCACertFile string
+	// DatabaseAWSRegion is an optional database cloud region e.g. when using AWS RDS.
+	DatabaseAWSRegion string
```

Comment:
Yeah, it's a part of "aws" database service configuration sub-field. I figured it'd make sense to have per-cloud configuration sub-sections to avoid potential conflicts/mixups (though we only support AWS now) and the flag/variable name reflects that.